graphql-java-support/src/main/java/com/apollographql/federation/graphqljava/FederationDirectives.java

```diff
@@ -168,7 +206,9 @@ private FederationDirectives() {}
                 externalDefinition,
                 requiresDefinition,
                 providesDefinition,
-                extendsDefinition)
+                extendsDefinition,
```

Comment:
Since I removed the `GraphQLDirective` for now (see [above](https://github.com/apollographql/federation-jvm/pull/144#discussion_r830073812)). I guess it's ok to not include them in `allDirectives`?

More generally, if there's a use case for `allDirectives` we should certainly add a matching test.

Reply:
@martinbonnin 
> Since I removed the `GraphQLDirective` for now (see [above](https://github.com/apollographql/federation-jvm/pull/144#discussion_r830073812)). I guess it's ok to not include them in `allDirectives`?

Yep, that sounds fine to me.

> More generally, if there's a use case for `allDirectives` we should certainly add a matching test.

I'm not sure of the use case, it was in the [initial commit for the repo](https://github.com/apollographql/federation-jvm/commit/f5a13182f76106d39d2674422abee8c599eea5a6#diff-d265303e57dcf25a6c597dde777163e1638e7bf140974a136afe87b8a300d36bR92) but it was unused then. I'm guessing it was meant to be a `GraphQLSchema`-equivalent for `allDefinitions`.